projects/experimental/components/tooltip/tooltip.template.html

```diff
@@ -0,0 +1,21 @@
+<span
```

Comment:
Why do we need this span and not just the `tui-icon`?

Reply:
made with `tuiIconButton`, but without `polymorpheusOutlet`